sdk/java/src/main/java/com/pulumi/docker/inputs/DockerBuildArgs.java

```diff
@@ -0,0 +1,347 @@
+// *** WARNING: this file was generated by pulumi-java-gen. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+package com.pulumi.docker.inputs;
+
+import com.pulumi.core.Either;
+import com.pulumi.core.Output;
+import com.pulumi.core.annotations.Import;
+import com.pulumi.core.internal.Codegen;
+import com.pulumi.docker.inputs.CacheFromArgs;
+import java.lang.Boolean;
+import java.lang.String;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import javax.annotation.Nullable;
+
+
+/**
+ * The Docker build context
+ * 
+ */
+public final class DockerBuildArgs extends com.pulumi.resources.ResourceArgs {
+
+    public static final DockerBuildArgs Empty = new DockerBuildArgs();
+
+    /**
+     * An optional map of named build-time argument variables to set during the Docker build. This flag allows you to pass built-time variablesthat can be accessed like environment variables inside the RUN instruction.
+     * 
+     */
+    @Import(name="args")
+    private @Nullable Output<Map<String,String>> args;
+
+    /**
+     * @return An optional map of named build-time argument variables to set during the Docker build. This flag allows you to pass built-time variablesthat can be accessed like environment variables inside the RUN instruction.
+     * 
+     */
+    public Optional<Output<Map<String,String>>> args() {
+        return Optional.ofNullable(this.args);
+    }
+
+    /**
+     * A cached image or list of build stages to use as build cache
+     * 
+     */
+    @Import(name="cacheFrom")
+    private @Nullable Output<Either<Boolean,CacheFromArgs>> cacheFrom;
+
+    /**
+     * @return A cached image or list of build stages to use as build cache
+     * 
+     */
+    public Optional<Output<Either<Boolean,CacheFromArgs>>> cacheFrom() {
+        return Optional.ofNullable(this.cacheFrom);
+    }
+
+    /**
+     * The path to the build context to use.
+     * 
+     */
+    @Import(name="context")
+    private @Nullable Output<String> context;
+
+    /**
+     * @return The path to the build context to use.
+     * 
+     */
+    public Optional<Output<String>> context() {
+        return Optional.ofNullable(this.context);
+    }
+
+    /**
+     * The path to the Dockerfile to use.
+     * 
+     */
+    @Import(name="dockerfile")
+    private @Nullable Output<String> dockerfile;
+
+    /**
+     * @return The path to the Dockerfile to use.
+     * 
+     */
+    public Optional<Output<String>> dockerfile() {
+        return Optional.ofNullable(this.dockerfile);
+    }
+
+    /**
+     * Environment variables to set on the invocation of docker build, for example to support DOCKER_BUILDKIT=1 docker build.
+     * 
+     */
+    @Import(name="env")
+    private @Nullable Output<Map<String,String>> env;
+
+    /**
+     * @return Environment variables to set on the invocation of docker build, for example to support DOCKER_BUILDKIT=1 docker build.
```

Comment:
Nit: curious seeing this docstring again, perhaps DOCKER_BUILDKIT=1 is no longer needed AFAIK, but if this docstring is coming from automated docs not our schema perhaps ok to ignore.